.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -48,11 +63,14 @@ stages:
         cp -r $(Pipeline.Workspace)/sonic-buildimage.${default_platform}/versions target/
         make freeze FREEZE_VERSION_OPTIONS=-r
         find files/build/versions 
-        for artifact in $artifacts
+        ordered_artifacts=$(echo "$artifacts" | grep -v -E "arm64|armhf" && echo "$artifacts" | grep -E "arm64|armhf")
```

Comment:
why armhf and arm64 are treated differently?

Reply:
We run 'make freeze' for amd64 at first. Then for other platform.
Because the number of target in amd64 is more than others, we use amd64 target to generate default version files.
Only If armhf/arm64 have different version requirement, we will create a arch-related version file.